bot/data/message_repository.py

```diff
@@ -68,3 +68,24 @@ async def check_message(self, message_id: int) -> bool:
         async with aiosqlite.connect(self.resolved_db_path) as db:
             async with db.execute('SELECT * FROM Messages WHERE id = ?', (message_id,)) as c:
                 return await c.fetchone() is not None
+
+    async def get_user_message_count(self, user_id, guild_id) -> int:
+        async with aiosqlite.connect(self.resolved_db_path) as db:
+            try:
+                c = await db.execute('SELECT count(*) FROM Messages WHERE fk_guildId = ? AND fk_authorId = ?', (guild_id, user_id,))
```

Comment:
use async with here so we dont have to worry about closing the connection

Reply:
async with asynchronously opens the connection and automatically closes after scope exits